science/plumed/files/0001-Added-enable-libsearch-to-configure.patch

```diff
@@ -0,0 +1,420 @@
+From fa0417f6dee0913e14501449839a38114b8f10af Mon Sep 17 00:00:00 2001
+From: Giovanni Bussi <giovanni.bussi@gmail.com>
+Date: Mon, 3 Apr 2017 10:53:46 +0200
+Subject: [PATCH 1/3] Added --enable-libsearch to configure
+
+This flags allow to enable/disable search for libraries.
+It is on by the fault, and it can be disabled using
+./configure --disable-libsearch
+This is useful when installing within package managers
+such as macports to make sure that only libraries explicitly
+listed as LDFLAGS are linked.
+---
+ configure    | 97 ++++++++++++++++++++++++++++++++++++++++++++++++++----------
+ configure.ac | 50 +++++++++++++++++++++++--------
+ 2 files changed, 119 insertions(+), 28 deletions(-)
+
+diff --git a/configure b/configure
+index eda363efc..30349ac3e 100755
+--- a/configure
++++ b/configure
+@@ -694,6 +694,7 @@ ac_subst_files=''
+ ac_user_opts='
+ enable_option_checking
+ enable_modules
++enable_libsearch
+ enable_doc
+ enable_pdfdoc
+ enable_debug
+@@ -1357,6 +1358,7 @@ Optional Features:
+   --enable-FEATURE[=ARG]  include FEATURE [ARG=yes]
+   --enable-modules        all/none/reset or : separated list such as
+                           +crystallization:-bias default: reset
++  --enable-libsearch      enable search for libraries, default: yes
```

Comment:
This is a strange approach. Take a look at
https://autotools.io/pkgconfig/pkg_check_modules.html
or some other macros or check how other software deals with searching packages.